### ISSUE TITLE
branch: screenSummary

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#58]
branch created to add screen summary elements for #57

**Comment by zepumph** [comment]
because of #62, I created `screenSummary` from the `screen-summary` branch. I'm not going to delete the hyphenated branch. Please use `screenSummary` from here on.

**Comment by twant** [comment]
This work was merged to master two weeks ago, and all looks good. Further work will continue on master.

**Comment by zepumph** [comment]
Should be re-deleted as part of phetsims/special-ops#198 and also screen-summary

**Comment by zepumph** [comment]
Deleted
